sql-plugin/src/main/scala/com/nvidia/spark/rapids/AvroDataFileReader.scala

```diff
@@ -436,6 +441,19 @@ class AvroDataFileReader(si: SeekableInput) extends AvroFileReader(si) {
     curBlockReady = false
   }
 
+  /**
+   * Skip the current block raw data to the given output stream.
```

Comment:
Comment is confused. You can refer to readNextRawBlock, as it read the data, but this is to skip the current raw block, aka, moves the pointer to the next block start

Reply:
done